### ISSUE TITLE
Support tls client certificates and kerberos auth

### ISSUE THREAD

**Issue by ibotty** [tylerholien/milena#17]
It would be great if milena supported user authentication, introduced in kafka 0.9.

**Comment by tylerholien** [comment]
I agree! I'm working on the new 0.9-style groups right now. Either auth or compression are next. This is a free-time only project right now, so sorry new features take so long. :disappointed:

**Comment by ibotty** [comment]
I don't want to demand anything. Great that you are doing the work. For my part that issue is mostly a keep-track-of-it.

BTW: I would appreciate if you could open issues (maybe with a "wip" label) when you start working on something. Even sketching the desired API (if you thought so far) would be a great help for others so they develop patches that are in line with your vision. (Of course I would love to see the progress in a "wip: 09-consumers" branch directly as well ;)))
